CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+Fortigate exporter uses GitHub to manage reviews of pull requests.
+
+* If you are a new contributor see: [Steps to Contribute](#steps-to-contribute)
+
+* If you have a trivial fix or improvement, go ahead and create a pull request,
+  addressing it to @bluecmd
+
+* If you plan to do something more impacting, first discuss your ideas
+  on our [IRC](https://matrix.to/#/#fortigate_exporter:matrix.org)
+  This will avoid unnecessary work and surely give you and us a good deal
+  of inspiration.
+
+* Relevant coding style guidelines are the [Go Code Review
+  Comments](https://code.google.com/p/go-wiki/wiki/CodeReviewComments)
+  and the _Formatting and style_ section of Peter Bourgon's [Go: Best
+  Practices for Production
+  Environments](https://peter.bourgon.org/go-in-production/#formatting-and-style).
+
+* Make sure you understand how Prometheus works [Prometheus Getting started](https://prometheus.io/docs/prometheus/latest/getting_started/)
+
+* Make sure created metrics do respect [Prometheus naming guidelines](https://prometheus.io/docs/practices/naming/)
+
+* Make sure your code respects the [Prometheus implementation guidelines](https://prometheus.io/docs/practices/instrumentation/#things-to-watch-out-for)
+
+## Steps to Contribute
+
+Should you wish to work on an issue, please claim it first by commenting on the GitHub issue that you want to work on it. This is to prevent duplicated efforts from contributors on the same issue.
+
+To setup your dev environment you need:
+- makefile binaries (we recommand you use linux or [cygwin](https://cygwin.com/install.html) to make things easier and freer)
```

Comment:
I would rather recommend the use of either Linux (capital L) or WSL. Cygwin is pretty.. poor.. compared to how well WSL works.
```suggestion
- A way to run Makefiles, either Linux or WSL is recommended
```

Reply:
You are right, I am not really convincing when I have to talk about Windows 😄. WSL is fine.